src/StudyViewer/StudyViewer.jsx

```diff
@@ -32,6 +34,25 @@ class StudyViewer extends React.Component {
     return (index === 0);
   }
 
+  exportToWorkspace = (buttonConfig) => {
+    this.setState({
+      exportToWorkspace: { ...buttonConfig },
+    });
+  };
+
+  exportingPFBToWorkspaceStateChange = (stateChange) => {
```

Comment:
some files call SingleStudyViewer directly, dont know how we could combine them, open to suggestions

Reply:
I'm not sure what would be the best way in js, i was thinking
- in `StudyViewer.jsx` define the functions outside of the class instead of inside
- in `SingleStudyViewer.jsx` do `import <function> from StudyViewer.jsx`

Not sure how to handle `this` 🙂
But the functions are short so it's not a big deal